lib/discourse_logstash_logger.rb

```diff
@@ -80,9 +80,9 @@ def add_with_opts(severity, message = nil, progname = nil, opts = {}, &block)
       #
       # In theory we could get logster to include the exception class and message in opts but logster currently does not
       # need those options so we are parsing it from the message for now and not making a change in logster.
-      if progname == "web-exception" && message =~ /^(\w+) \((.+)\)\n/
+      if progname == "web-exception" && message =~ /\A([^\(\)]+)\s{1}\(([\s\S]+)\)/
```

Comment:
What's an example of a message we might match here? It looks like there's a `A` but I don't see it in the tests 🤔

Reply:
O this isn't an `A`. `\A` means `start of string` whereas `^` means `start of line`. What we want here is start of string and start of line as there can be multiple lines in a string.

Reply:
Oops!

Reply:
<img width="634" alt="Screenshot 2024-07-10 at 11 00 20 AM" src="https://github.com/discourse/discourse/assets/1555215/f9d7b947-420b-4768-9a9f-0198de32f136">

regexr bamboozled me